frontend/vite.web.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import {resolve} from "path";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+    plugins: [react()],
+    build: {
+        rollupOptions: {
+            input: {
+                main: resolve(__dirname, 'student.html'),
+                studio: resolve(__dirname, 'studio.html'),
+            }
+        }
+    },
+    resolve: {
+        alias: [
+            {
+                // this is required for the SCSS modules
+                find: /^~(.*)$/,
+                replacement: '$1',
+            },
+        ],
+    },
+})
```

Comment:
This config file is used simply to run a dev server so you can do quick iterative development without constantly rebuilding and reloading the LMS or Studio UI.